### DISCUSSION TITLE
[2021.10.20] 동물보호소에서 중성화한 동물

### DISCUSSION THREAD

**Discussion by chaeyeon4u** [pyunji/Hyundai-ITnE-Team_QuadCore#150] [SQL]
[동물보호소에서 중성화한 동물](https://programmers.co.kr/learn/courses/30/lessons/59045)

**Comment by chaeyeon4u** [comment]
[동물보호소에서 중성화한 동물 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59045.sql)

**Comment by pyunji** [comment]
[동물보호소에서 중성화한 동물 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59045.sql)
